services/editor/src/store/ducks/selectedKey.js

```diff
@@ -295,33 +294,84 @@ export function saveKey() {
   };
 }
 
-const deleteKeyAlert = key => ({
+const deleteKeyAlert = (key, aliases = []) => ({
   title: 'Warning',
-  message: `Are you sure you want to delete '${key}' key?`,
+  message: `Are you sure you want to delete '${key}'?${
+    aliases.length ? `\nAll aliases will also be deleted:\n${aliases.join('\n')}` : ''
+  }`,
 });
 
 export function deleteKey() {
   return async function (dispatch, getState) {
-    const { selectedKey: { key } } = getState();
+    const { selectedKey: { key, aliases } } = getState();
 
-    if (!(await dispatch(showConfirm(deleteKeyAlert(key)))).result) return;
+    if (!(await dispatch(showConfirm(deleteKeyAlert(key, aliases)))).result) return;
 
     dispatch(push('/keys'));
     try {
       await fetch(`/api/keys/${key}`, {
```

Comment:
Is it ok for DELETE to get body data?

Reply:
yes
https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request